SimplePolicy/Settings.cs

```diff
@@ -24,6 +26,8 @@ public Settings(IConfigurationSection section)
         {
             this.MaxTransactionsPerBlock = GetValueOrDefault(section.GetSection("MaxTransactionsPerBlock"), 500, p => int.Parse(p));
             this.MaxFreeTransactionsPerBlock = GetValueOrDefault(section.GetSection("MaxFreeTransactionsPerBlock"), 20, p => int.Parse(p));
+            this.MaxFreeTransactionSize = GetValueOrDefault(section.GetSection("MaxFreeTransactionSize"), 400, p => int.Parse(p));
```

Comment:
Why this is 400 instead of 1024 as in config?

Reply:
good catch, should be 1024 I think.

Reply:
I think we just forgot to change the default value, fixed. thanks.